Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM odise/busybox-node
```

Comment:
I think that it would be better to base the image on the well-known, community-maintained dockerfile/nodejs image from http://dockerfile.github.io .